.github/workflows/auto-reqs-update-pr.yml

```
@@ -15,7 +15,7 @@ jobs:
           [[ ${GITHUB_REF} =~ $regex ]]
           pr_repo=$BASH_REMATCH
           # find other PRs from that repo
-          other_prs=$(curl -s ${renku_repo}/pulls | jq ".[] | select(.head.ref | test(\"auto-update/${pr_repo}\")) | .number")
+          other_prs=$(curl -s ${renku_repo}/pulls | jq ".[] | select((.head.ref | test(\"auto-update/${pr_repo}\")) and (.head.ref != \"${GITHUB_REF:11}\")) | .number")
```

```suggestion
          other_prs=$(curl -s ${renku_repo}/pulls | jq ".[] | select((.head.ref | test(\"auto-update/${pr_repo}\")) and (.head.ref != \"${GITHUB_REF:12}\")) | .number")
```
I think you also have to remove the slash.

---

ah thanks, in the test string I had I accidentally removed the first slash...

---

turns out I had it right :) the counting starts at zero

```
> echo $REF
refs/heads/auto-update/renku-testing-1234
>  echo ${REF:11}
```

---

aaa, sorry - I thought you wanted to cut the `auto-update/`, from `auto-update/renku-testing-1234` which wouldn't make much sense. Sorry, my bad.

---

no worries, I'm usually off by one on these sorts of things :)